config/clients/java/template/config-Configuration.java.mustache

```diff
@@ -40,9 +41,10 @@ public class Configuration implements BaseConfiguration {
      */
     @Override
     public void assertValid() throws FgaInvalidParameterException {
-        if (this.apiUrl == null) {
-            throw new FgaInvalidParameterException(
-                    "apiUrl", "Configuration", new NullPointerException("API URL cannot be null."));
+        if (isNullOrWhitespace(apiUrl)) {
+            // If apiUrl is null/empty/whitespace it will resolve to
+            // DEFAULT_API_URL when getApiUrl is called.
```

Comment:
Just FYI this would not work when you are validating more than the API URL, you won't be able to return

Reply:
Pref have this all in an if section instead of returning

Reply:
What do you think of this? [5e09af7](https://github.com/openfga/sdk-generator/pull/173/commits/5e09af76d1d1a75774ba8d211c09888d4bd8448b)